src/libraries/System.Data.Odbc/ref/System.Data.Odbc.cs

```diff
@@ -259,6 +262,7 @@ public sealed partial class OdbcParameter : System.Data.Common.DbParameter, Syst
         [System.ComponentModel.DefaultValueAttribute(System.Data.Odbc.OdbcType.NChar)]
         [System.Data.Common.DbProviderSpecificTypePropertyAttribute(true)]
         public System.Data.Odbc.OdbcType OdbcType { get { throw null; } set { } }
+        public int Offset { get { throw null; } set { } }
```

Comment:
This should not be added see https://github.com/dotnet/runtime/blob/master/src/libraries/System.Data.Odbc/src/MatchingRefApiCompatBaseline.txt